dev/PushNotifications/PushNotificationChannel.cpp

```diff
@@ -148,6 +183,44 @@ namespace winrt::Microsoft::Windows::PushNotifications::implementation
     }
     CATCH_RETURN()
 
+    HRESULT __stdcall PushNotificationChannel::OnRawNotificationReceived(unsigned int payloadLength, _In_ byte* payload, _In_ HSTRING /*correlationVector */) noexcept
+    {
+        try
+        {
+            BOOL foregroundHandled = false;
+            THROW_IF_FAILED(InvokeAll(payloadLength, payload, &foregroundHandled));
```

Comment:
foregroundHandled should be used along with channelInfo != nullptr to determine protocol launch